src/test/java/edu/harvard/iq/dataverse/dataverse/banners/dto/BannerMapperTest.java

```diff
@@ -0,0 +1,98 @@
+package edu.harvard.iq.dataverse.dataverse.banners.dto;
+
+import com.google.common.collect.Lists;
+import com.google.common.collect.Sets;
+import edu.harvard.iq.dataverse.Dataverse;
+import edu.harvard.iq.dataverse.dataverse.banners.DataverseBanner;
+import edu.harvard.iq.dataverse.dataverse.banners.DataverseLocalizedBanner;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.time.LocalDateTime;
+import java.util.Date;
+
+import static edu.harvard.iq.dataverse.util.DateUtil.convertToDate;
+
+public class BannerMapperTest {
+
+    private static final Date FROM_TIME = convertToDate(
+            LocalDateTime.of(2018, 10, 1, 9, 15, 45));
+    private static final Date TO_TIME = convertToDate(
+            LocalDateTime.of(2018, 11, 2, 10, 25, 55));
+    private static final Path BANNER_PATH = Paths.get("src/test/resources/images/banner.png");
+
+    @Test
+    public void shouldMapToDto() throws IOException {
+        //given
+        BannerMapper bannerMapper = new BannerMapper();
+        DataverseBanner dataverseBanner = createBannerEntity();
+
+        //when
+        DataverseBannerDto dto = bannerMapper.mapToDto(dataverseBanner);
+        DataverseLocalizedBannerDto bannerDto = dto.getDataverseLocalizedBanner().get(0);
+        DataverseLocalizedBanner localizedBanner = dataverseBanner.getDataverseLocalizedBanner().iterator().next();
```

Comment:
Very confusing naming of variables.
What do you think of:
`dto` -> `bannerDto`
`bannerDto` -> `localizedBannerDto`